Gemfile

```diff
@@ -8,6 +8,13 @@ group :development do
   gem 'pry'
 end
 
+if RUBY_VERSION < "2.0"
+  gem 'json', '< 2.0.0'
```

Comment:
It can be `~> 1.8` too, which would work with 1.9 -  I don't see why leaving any room for users to be the target of a serious CVE. If someone has a legitimate reason to still use 1.7 after that, let them open a bug and we'll handle it